_data/simple-icons.json

```diff
@@ -6135,9 +6135,9 @@
         },
         {
             "title": "Hyundai",
-            "hex": "002C5F",
-            "source": "https://en.wikipedia.org/wiki/File:Hyundai_Motor_Company_logo.svg",
-            "guidelines": "https://www.hyundai.pl/fileadmin/user_upload/media/logo/201607_HYU_Guideline_ENG_small.pdf"
+            "hex": "002C5E",
+            "source": "https://www.hyundainews.com",
+            "guidelines": "https://www.hyundainews.com/en-us/contact"
```

Comment:
```suggestion
            "source": "https://www.hyundai.com"
```

SVG icon available as part of the webfont used on their global homepage. Also don't think a contact page is an adequate guidelines link.

Reply:
![image](https://github.com/simple-icons/simple-icons/assets/35000807/0f701bfe-e6fd-41d9-bf60-f962306badc2)
This is why I attached the contact page as that is how they want media inquiries to be made.

Reply:
Corrected the source and updated the icon